eyre/tests/test_option.rs

```diff
@@ -0,0 +1,15 @@
+mod common;
+
+use self::common::maybe_install_handler;
+use eyre::OptionExt;
+
+#[test]
+fn test_option_or_eyre() {
```

Comment:
```suggestion
fn test_option_ok_or_eyre() {
```

Should fixing this name be worth another workflow run?